packages/website/docs/FlatSelectConnected.md

```diff
@@ -0,0 +1,28 @@
+# Best Practices
+
+The Flat select option should be used for single choice selection.
+
+The width of each of the options should be the same, we don't want any of the options feel more important than the other.
```

Comment:
@johnnyCoveo Does that mean that the width of the flat select should not change depending on the lenght of the text selected? 🤔

Reply:
Yes, all options should have the same width!